helm/capi-migration/Chart.yaml

```diff
@@ -3,4 +3,6 @@ name: capi-migration
 description: capi-migration chart.
 home: https://github.com/giantswarm/capi-migration
 version: [[ .Version ]]
-appVersion: [[ .AppVersion ]]
+appVersion: '[[ .AppVersion ]]'
+annotations:
+  config.giantswarm.io/version: add-capi-migration
```

Comment:
is `add-capi-migration` the desired value here? I expected to have something dynamic similar to `[[ .AppVersion ]]` above

Reply:
`add-capi-migration` is the name of the branch there: https://github.com/giantswarm/config/pull/35

Eventually it should become `1.x.x` when this branch is merged. I'm not sure if I should already add this app to the config repo?

Reply:
ok now I understood.
This all will eventually be a production app and therefore it makes sense to have it already in config operator like all other apps and not just in a branch IMHO.
No strong opinion though

Reply:
```suggestion
  config.giantswarm.io/version: 1.x.x
```